transport/serial/usb.go

```diff
@@ -12,7 +12,11 @@ type usbDevice struct {
 }
 
 var knownDevices = []usbDevice{
-	{VID: "239A", PID: "8029"}, // rak4631_19003
+	// rak4631_19003
+	{VID: "239A", PID: "8029"},
```

Comment:
Thanks for adding these. I'd like to probably move this magic connection stuff somewhere else and make it more explicit as to what is happening, but it sure is nice in the meantime 😆